### PR TITLE
Fix delivery time labels not displayed

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.4.x
| Description?  | Fix error with label and placeholder in product page + code refactoring in forms.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-5479
| How to test?  | In the product page in BO (shipping tab) check the delivery time's labels .

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9044)

### CODE REVIEW

**Comment by eternoendless** [comment]
:lgtm:

---

Reviewed 15 of 15 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9044#-:-LCDelue8qG8_YjNwKtJ:bnfp4nl)*

**Review by eternoendless — DISMISSED** [review]

**Comment by marionf** [comment]
@PierreRambaud 

When I click on the "edit" link next to the "Default delivery time"

capture du 2018-05-11 12-17-07

I have a 404 because it's not the link of the symfony page but of the legacy one

capture du 2018-05-11 12-17-58

**Comment by PierreRambaud** [comment]
Nice catch @marionf don't know why I don't try this link (just try to change and save --').

File: src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/PerformanceFormHandler.php [hunk 1]
```diff
@@ -55,8 +55,7 @@ public function __construct(
         FormFactoryInterface $formFactory,
         FormDataProviderInterface $formDataProvider,
         CombinationFeature $combinationFeature
-    )
-    {
+    ) {
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/PerformanceFormHandler.php:58]: something is wrong here, maybe an error with a rebase or a merge between branches, but AbstractFormHandler class doesnt exists.

Comment (mickaelandrieu) [reply src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/PerformanceFormHandler.php:58]: nevermind this class is deprecated and we're not using it anymore.

Comment (PierreRambaud) [reply src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/PerformanceFormHandler.php:58]: I just check for all ChoiceType types, and doing my boy scout on checked files

**Comment by eternoendless** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9044)*

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @PierreRambaud

**Pull request merged**